sphinx/api/configuration/file-creation.rst

```diff
@@ -116,6 +116,11 @@ root directory to install file targets with a relative path:
          ``~/.ssh/config`` to specify the user ID for logging into ``HOST``
          if required.
 
+      .. note::
+
+         * When installing files, broken symbolic links are rejected.
+         * When installing directories using Git or Rsync broken symlinks
```

Comment:
Remove "using Git or Rsync" - it applies in all cases (including svn).
Also add comma.